howto-gallery/scripts/CreateExplosion.ts

```diff
@@ -46,12 +46,12 @@ export function createFireAndWaterExplosions(
 ) {
   const overworld = mc.world.getDimension("overworld");
 
-  const explosionLoc = { x: targetLocation.x + 0.5, y: targetLocation.y + 0.5, z: targetLocation.z + 0.5};
+  const explosionLoc = { x: targetLocation.x + 0.5, y: targetLocation.y + 0.5, z: targetLocation.z + 0.5 };
```

Comment:
Great place to add examples using @minecraft/math !

Reply:
Updated